src/Automation/Automation.Test/ScenarioTests/UpdateManagementTests.ps1

```diff
@@ -424,3 +424,101 @@ function Test-CreateWindowsMonthlySoftwareUpdateConfiguration() {
+Test-CreateWindowsIncludeKbNumbersSoftwareUpdateConfiguration
+#>
+function Test-CreateWindowsIncludeKbNumbersSoftwareUpdateConfiguration() {
+
+    $aa = "Automate-d2b38167-d3ca-4d1f-a020-948eee21b6bc-EJP"
+	$rg = "DefaultResourceGroup-EJP"
+    $azureVMIdsW = @(
+	   "/subscriptions/d2b38167-d3ca-4d1f-a020-948eee21b6bc/resourceGroups/ikwjp12r201-RG/providers/Microsoft.Compute/virtualMachines/ikwjp12r201"
+	)
+
+    $name = "mo-monthly-01"
+    $startTime = ([DateTime]::Now).AddMinutes(10)
+    $duration = New-TimeSpan -Hours 2
+	$s = New-AzAutomationSchedule -ResourceGroupName $rg `
+                                       -AutomationAccountName $aa `
+                                       -Name $name `
+                                       -Description test-OneTime `
+                                       -MonthInterval 1 `
+                                       -DaysOfMonth Two,Five `
+                                       -StartTime $startTime `
+                                       -ForUpdate
+
+    $suc = New-AzAutomationSoftwareUpdateConfiguration  -ResourceGroupName $rg `
+                                                             -AutomationAccountName $aa `
+                                                             -Schedule $s `
+                                                             -Windows `
+                                                             -AzureVMResourceId $azureVMIdsW `
+                                                             -Duration $duration `
+                                                             -IncludedKbNumber @("KB-01", "KB-02")
+
+    Assert-NotNull $suc "New-AzAutomationSoftwareUpdateConfiguration returned null"
+    Assert-AreEqual $suc.Name $name "Name of created software update configuration didn't match given name"
+    Assert-NotNull $suc.UpdateConfiguration "UpdateConfiguration of the software update configuration object is null"
+    Assert-NotNull $suc.ScheduleConfiguration "ScheduleConfiguration of the software update configuration object is null"
+    Assert-AreEqual $suc.ProvisioningState "Provisioning" "software update configuration provisioning state was not expected"
+    Assert-AreEqual $suc.UpdateConfiguration.OperatingSystem "Windows" "UpdateConfiguration Operating system is not expected"
+    Assert-AreEqual $suc.UpdateConfiguration.Duration $duration "UpdateConfiguration Duration is not expected"
+    Assert-AreEqual $suc.UpdateConfiguration.AzureVirtualMachines.Count 1 "UpdateConfiguration created doesn't have the correct number of azure virtual machines"
+    Assert-AreEqual $suc.UpdateConfiguration.NonAzureComputers.Count 0 "UpdateConfiguration doesn't have correct value of NonAzureComputers"
+    Assert-NotNull $suc.UpdateConfiguration.Windows "Linux property of UpdateConfiguration is null"
+    Assert-AreEqual $suc.UpdateConfiguration.Windows.IncludedKbNumbers.Count 2
+    Assert-AreEqual $suc.UpdateConfiguration.Windows.IncludedKbNumbers[0] "KB-01"
```

Comment:
UX input validation for this is that it must be a number (not including the 'KB' text). Not sure off the top of my head if the web service checks too. But ideally as an example, it should just be a number.

Reply:
There is no validation in webservice. Fixed the test